sunpy/net/solarnet/attrs.py

```diff
@@ -0,0 +1,58 @@
+from sunpy.net._attrs import Detector, Time, Wavelength
+from sunpy.net.attr import AttrAnd, AttrOr, AttrWalker, DataAttr, SimpleAttr
+
+
+class Datasets(SimpleAttr):
+    """
+    Name of datasets
+    """
+
+class limit(SimpleAttr):
+    """
+    number of links default value(20)
```

Comment:
```suggestion
    Sets the max number of results, defaults to 20.
```
This default is set server side?

Reply:
yes